integration/test/performance_tracker/performance_query_utils.go

```diff
@@ -0,0 +1,78 @@
+package data_collector
+
+import (
+	"time"
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+)
+
+const (
+	Namespace = "CWAgent"
+	DimensionName = "InstanceId"
+	Stat = "Average"
+	Period = 30
+)
+
+// CWGetMetricDataAPI defines the interface for the GetMetricData function
+type CWGetMetricDataAPI interface {
+	GetMetricData(ctx context.Context, params *cloudwatch.GetMetricDataInput, optFns ...func(*cloudwatch.Options)) (*cloudwatch.GetMetricDataOutput, error)
+}
```

Comment:
I don't think we need t his. We can just invoke the SDK client directly and cut out needing to define this interface just to pass the client in to the `GetMetrics` function.

Reply:
removed